mobile_verifier/src/reward_shares.rs

```diff
@@ -134,8 +244,7 @@ pub fn get_scheduled_tokens(
 ) -> Result<Decimal, ClockError> {
     (*GENESIS_START <= start)
         .then(|| {
-            (Decimal::from(GENESIS_REWARDS_PER_DAY)
-                / Decimal::from(Duration::hours(24).num_seconds()))
+            (GENESIS_REWARDS_PER_DAY / Decimal::from(Duration::hours(24).num_seconds()))
```

Comment:
Should you not be using TOTAL_EMISSIONS_POOL * 0.4 here ? not sure i understand why both TOTAL_EMISSIONS_POOL and GENESIS_REWARDS_PER_DAY is defined

Also suggest renaming `get_scheduled_tokens` and `get_scheduled_dc_tokens`

Reply:
i concur on the use of both; i believe once we are issuing rewards from the emissions pool we should no longer be issuing rewards from the genesis period

Reply:
I think we should start with the 30B total and work things out from there to the daily or epoch count